src/main/java/com/example/basicbookstoreprojectnew/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.basicbookstoreprojectnew.controller;
+
+import com.example.basicbookstoreprojectnew.dto.BookDto;
+import com.example.basicbookstoreprojectnew.dto.CreateBookRequestDto;
+import com.example.basicbookstoreprojectnew.model.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
to avoid setting "api" for every controller, add `server.servlet.context-path=/api` to `application.properties`
```suggestion
@RequestMapping("/books")
```